### DISCUSSION TITLE
What happened to #6448 [HELP NEEDED] Major planned/missing features?

### DISCUSSION THREAD

**Discussion by GfEW** [TeamNewPipe/NewPipe#10958] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

There used to be [#6448](https://web.archive.org/web/20230712131653/https://github.com/TeamNewPipe/NewPipe/issues/6448), providing a list of major planned/missing features for which help was needed. 

As of now, it gives a [404](https://github.com/TeamNewPipe/NewPipe/issues/6448). Why is that? 

Shouldn't it be **closed** with some explanation (be it _completed_ if all of its main points have been accomplished, or _wontfix_, or moved, split up or whatever is appropriate) **rather than just vanish**? 

Deleting the entire issue seems questionably intransparent, it makes some part of NewPipe's development process, including any discussion thereof, unnecessarily difficult to comprehend.

**Comment by wb9688** [comment, marked as answer]
That issue was made by our team member @opusforlife2, who was shadow banned by GitHub, hence his issues disappeared. We unfortunately can't do much about that and he has already contacted GitHub to resolve it, but not yet received a reply from GitHub.

**Reply by GfEW** [reply]
Huh, weird stuff. 
Thanks for the info though.

p. s. @wb9688: Is there any place that elaborates on the background?
